requirements.txt

```diff
@@ -1,1591 +1,50 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.10
```

Comment:
Same comment here as for `api/runtime/requirements.txt` - can you please use Python 3.7?